spec/features/idv/analytics_spec.rb

```diff
@@ -850,6 +850,10 @@ def wait_for_event(event, wait)
         to receive(:biometric_comparison_required?).
         and_return({ biometric_comparison_required: true })
 
+      allow_any_instance_of(DocAuth::Response).to receive(:selfie_status).and_return(:success)
```

Comment:
I know other devs have campaigned against using `allow_any_instance_of` in testing..... I'm not asking you to change it here, just flagging it as something that I've seen come up in the past.